plugins/aws/connect/instanceExportedReportsEncrypted.js

```diff
@@ -0,0 +1,134 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Connect Instance Exported Reports Encrypted',
+    category: 'Connect',
+    domain: 'Content Delivery',
+    description: 'Ensure that Amazon Connect instances have encryption enabled for exported reports being saved on S3.',
+    more_info: 'You can configure Amazon Connect instance to save exported reports to be saved on S3. When you save ' +
+        'such data on S3, enable encryption for the data and use a KMS key with desired encrypted level to meet regulatory compliance requirements within your organization.',
+    link: 'https://docs.aws.amazon.com/connect/latest/adminguide/encryption-at-rest.html',
+    recommended_action: 'Modify Connect instance data storage configuration and enable encryption for exported reports',
+    apis: ['Connect:listInstances', 'Connect:listInstanceExportedReportsStorageConfigs', 'KMS:listKeys', 'KMS:describeKey'],
+    settings: {
+        connect_exported_reports_encryption_level: {
+            name: 'Connect exported reports Encryption Level',
```

Comment:
The initials should be capital
Connect Exported Reports Target Encryption Level